src/Miso/Effect.hs

```diff
@@ -87,6 +87,13 @@ batch actions = sequence_
   | action <- actions
   ]
 -----------------------------------------------------------------------------
+-- | Like @batch@ but action are discarded
```

Comment:
Here it might not be clear what actions are being discarded. It's the results of the JSM computations, right?

```suggestion
-- | Like @batch@ but actions (results of the JSM computations) are not fed back to the 'update'.
```